apps/client/src/common/components/edit-form-drawer/constants.ts

```diff
@@ -0,0 +1,214 @@
+import { ParamField } from './types';
+
```

Comment:
There is probably no way for you to have known this, but there is a field missing in some views.
All views apart from `MinimalTimer` call formatTime which will allow the user to declare whether the times should be in 12 or 24 hour format
https://github.com/cpvalente/ontime/blob/master/apps/client/src/common/utils/time.ts#L29

Reply:
Even `Countdown`?

Reply:
yes, even countdown. The `formatTime` function is called for `startTime` and `endTime `fields

Reply:
That means all views should have the edit button then, right?

Reply:
that is correct

Reply:
Addressed with https://github.com/cpvalente/ontime/pull/420/commits/ab8b2005ebf9bc02dbc879e769c18c8269f662f6 & https://github.com/cpvalente/ontime/pull/420/commits/48b5d20b955c334247923b081e2db50db0754132